src/Orchard.Cms.Web/Modules/Orchard.Setup/Assets/js/strength.js

```diff
@@ -21,7 +21,7 @@
         var upperCase = new RegExp('[A-Z]');
         var lowerCase = new RegExp('[a-z]');
         var number = new RegExp('[0-9]');
-        var specialchar = new RegExp('([!,%,&,@,#,$,^,*,?,_,~])');
+        var specialchar = new RegExp('([!,%,&,@,#,$,^,*,?,_,~,",�,(,)])');
```

Comment:
seriously though, it does pose the question which chars to support and which ones not to.